app/forms/sections/items/ItemPackagingSealTypeFormProvider.scala

```diff
@@ -33,13 +34,13 @@ class ItemPackagingSealTypeFormProvider @Inject() extends Mappings {
       mapping(
         packagingSealTypeField ->
           text(sealTypeRequiredErrorKey)
-            .verifying(maxLength(maxLengthTextBoxValue, sealTypeLengthErrorKey))
+            .verifying(maxLength(maxLengthSealTypeField, sealTypeLengthErrorKey))
```

Comment:
Alphanumeric check? i.e. user enters just `!` or `???`

Reply:
As discussed, will leave it off this one. And only add to the textarea to follow the same pattern on other text area pages.